tests/unittests/GraphOptzTest.cpp

```diff
@@ -84,6 +84,14 @@ static Function *optimizeFunction(Function *F) {
   return G;
 }
 
+template <typename NodeT = Node>
```

Comment:
Please add documentation explaining what this function does. Look at other static helper functions for examples.

Reply:
thanks, added